features/claims/scheme_ten/advocate_trial_claim_submission.feature

```diff
@@ -50,8 +51,9 @@ Feature: Advocate submits a claim for a final fee trial case under scheme 10
 
     Then I click "Continue" in the claim form
 
-    And I upload 3 documents
-    And I check the boxes for the uploaded documents
+    And I upload 1 document
```

Comment:
change to upload specific doc and test on summary page [here](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2434/commits/11f3eb5b3529095fb7d895bc39f023b396102cd6#diff-9b5d9de3a7d6bfcdfc71965ed1d786f3R78)